docs/en_us/platform_api/source/mobile/course_info.rst

```diff
@@ -1,10 +1,8 @@
 ##################################################
-Course Information API Module
+Platform Mobile Course Information API Module
```

Comment:
This and following headings are a bit confusing, because they are structured slightly differently in some cases. 
Should this be "EdX Platform Mobile API Course Information Module"? i.e. a module within the Mobile API?
However, we refer to "platform mobile course information API" below.